languagetool-language-modules/pt/src/main/java/org/languagetool/language/BrazilianPortuguese.java

```diff
@@ -54,4 +75,11 @@ public String[] getCountries() {
     return new String[]{"BR"};
   }
 
+  public static @NotNull BrazilianPortuguese getInstance() {
+    Language language = Objects.requireNonNull(Languages.getLanguageForShortCode(LANGUAGE_SHORT_CODE));
+    if (language instanceof BrazilianPortuguese brazilianPortuguese) {
```

Comment:
This will not work with the close source part :( See my comments in German: https://github.com/languagetool-org/languagetool/blob/482657017ccde7123292c801ef9f332a4f6c9e80/languagetool-language-modules/de/src/main/java/org/languagetool/language/GermanyGerman.java#L90

This applies for those languages and their variants:

de
en
pt